README.md

```diff
@@ -108,16 +108,25 @@ opened where you can further experiment with the test.
 * `page.driver.scroll_to(left, top)` Scroll to a given position.
 * `element.send_keys(*keys)` Send keys to a given node.
 
+## Set user agent
+
+To change the user-agent of the driver for all following requests, use the `user-agent` browser option:
+
+```
+Capybara::Cuprite::Driver.new(app, { browser_options: { 'user-agent': 'Your own user agent string' } })
+```
+
+If you want to change the user-agent for one request, you can use the request headers.
```

Comment:
:thinking: this spec is passed:
```
      it "supports User-Agent" do
        @driver.add_header("User-Agent", "foo")
        @session.visit "/"
        expect(@driver.evaluate_script("window.navigator.userAgent")).to eq("foo")
        @session.visit "/"
        expect(@driver.evaluate_script("window.navigator.userAgent")).to eq("foo")
      end
```
can you please provide the version of which you using and could you please provide a test with this behavior reproduction? It's probably it is a bug

Reply:
I don't have the setup running anymore, because we stopped using Cuprite due to incompatibility with our test suite. The request headers through `add_header` works okay, but is reset after each spec. We set the user agent in the driver before starting the suite, therefore we lost the user agent header. Only when providing the `browser_options` you can persist the user agent for all requests in the whole suite.